cardano-node/src/Cardano/Node/Tracing/Tracers/NodeToClient.hs

```diff
@@ -213,6 +222,152 @@ docTChainSync = Documented [
         \busy states are where the server is expected to send a reply."
   ]
 
+--------------------------------------------------------------------------------
+-- LocalTxMonitor Tracer
+--------------------------------------------------------------------------------
+
+severityTTxMonitor
+  :: TraceLabelPeer peer
+       (TraceSendRecv (LTM.LocalTxMonitor (GenTxId blk) (GenTx blk) SlotNo))
+  -> SeverityS
+severityTTxMonitor _ = Info
+
+namesForTTxMonitor
+  :: TraceLabelPeer peer
+       (TraceSendRecv (LTM.LocalTxMonitor (GenTxId blk) (GenTx blk) SlotNo))
+  -> [Text]
+namesForTTxMonitor (TraceLabelPeer _ v) = namesForTTxMonitor' v
+  where
+    namesForTTxMonitor' (TraceSendMsg msg) = "Send"    : namesForTTxMonitor'' msg
+    namesForTTxMonitor' (TraceRecvMsg msg) = "Receive" : namesForTTxMonitor'' msg
+
+    namesForTTxMonitor'' (AnyMessageAndAgency _agency msg) = namesForTTxMonitor''' msg
+
+    namesForTTxMonitor''' :: Message (LTM.LocalTxMonitor (GenTxId blk) (GenTx blk) SlotNo)
+                                     from to
+                          -> [Text]
+    namesForTTxMonitor''' LTM.MsgAcquire {}       = ["Acquire"]
```

Comment:
Please don't align code

Reply:
That's done in the whole module it will look inconsistent if we do that.

Reply:
If we leave the aligned code it will further encourage others to do so. I'm trying to prevent additional alignment of code like this. It's ok if it looks inconsistent as in the future I will have to undo alignment in the library.

Reply:
Though is this a problem, usually when submitting a PR which needs to re-align some of the code, one can filter out space only changes.  Most tools allow that, including github UI.  Or maybe, there's something else to it?

Reply:
The problem is future changes to one line of the aligned code can potentially introduce semantically null changes in all of the surrounding code that needs to be re-aligned.

Reply:
Indeed, one can filter  out space only changes by [ticking a box](https://github.blog/2018-05-01-ignore-white-space-in-code-review/) in github diff tool.  The same in various editors.

Reply:
but if you still want unaligned code it's up to you

Reply:
I'd like to keep the code formatting in `cardano-node` as consistent as possible and as such we have a  `STYLE.md`: 
> Indentation
    Do not vertically align code. Mainly because the need to change identifiers and re-aligning the other lines causes semantically null changes in the diff.

Reply:
Fixed